### PR TITLE
Add quotes and company logos.

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
Permission has been granted for these to be used by GitHub. I will CC the various people quoted just so they are aware on the PR.

### CODE REVIEW

File: app/views/home/businesses.html.erb [hunk 1]
```diff
@@ -97,41 +97,41 @@
...
         <div>
           <div class="d-table-cell v-align-top pr-3">
-            <img src="https://api.adorable.io/avatars/285/ossfriday.png" alt="" width="50" height="50" style="border-radius:25px;">
+            <img src="https://avatars1.githubusercontent.com/u/691109?v=3&s=100" alt="" width="50" height="50" style="border-radius:25px;">
           </div>
           <blockquote class="d-table-cell v-align-top text-gray">
-            <p class="alt-mono-font alt-text-small">“Quote that talks about how some amazing company invests in open source.”</p>
-            <footer class="alt-text-small">Jane Smith, Tech Giant</footer>
+            <p class="alt-mono-font alt-text-small">Open source aligns with our mission to make the world more open and connected. Working with the community gives us an opportunity to share our work and the challenges we’re aiming to solve, and encourages talented people to contribute their ideas and effort to move the world forward.</p>
+            <footer class="alt-text-small">Christine Abernathy (<a href="https://github.com/caabernathy">@caabernathy</a>), Open Source at <a href="https://github.com/facebook">Facebook</a></footer>
```
Comment (MikeMcQuaid) [inline app/views/home/businesses.html.erb]: @caabernathy Just a heads up that your quote is being used here. Let me know either here or at mikemcquaid@github.com if you have any problem with that. Ignore this if you're 👍. Thanks!

File: app/views/home/businesses.html.erb [hunk 2]
```diff
@@ -97,41 +97,41 @@
...
         <div>
           <div class="d-table-cell v-align-top pr-3">
-            <img src="https://api.adorable.io/avatars/285/ossfriday2.png" alt="" width="50" height="50" style="border-radius:25px;">
+            <img src="https://avatars0.githubusercontent.com/u/1112?v=3&s=100" alt="" width="50" height="50" style="border-radius:25px;">
           </div>
           <blockquote class="d-table-cell v-align-top text-gray">
-            <p class="alt-mono-font alt-text-small">“Another quote that talks about how Open Source Friday saved my business and my dog.”</p>
-            <footer class="alt-text-small">Erica McGitterson, Bootstraps R Us</footer>
+            <p class="alt-mono-font alt-text-small">Contributing to open source projects continues to be one of the most popular 20% projects for Google engineers. Open source plays such an important role in so much of what we do, so we try to make <a href="https://opensource.google.com/docs/patching/">the process</a> as easy as possible.</p>
+            <footer class="alt-text-small">Will Norris (<a href="https://github.com/willnorris">@willnorris</a>), Open Source at <a href="https://github.com/google">Google</a></footer>
```
Comment (MikeMcQuaid) [inline app/views/home/businesses.html.erb]: @willnorris Just a heads up that your quote is being used here. Let me know either here or at mikemcquaid@github.com if you have any problem with that. Ignore this if you're 👍. Thanks!

File: app/views/home/index.html.erb [hunk 3]
```diff
@@ -159,25 +159,25 @@
...
           <div class="d-table-cell v-align-top pr-3">
-            <img src="https://api.adorable.io/avatars/285/ossfriday.png" alt="" width="50" height="50" style="border-radius:25px;">
+            <img src="https://avatars2.githubusercontent.com/u/9155255?v=3&s=100" alt="" width="50" height="50" style="border-radius:25px;">
           </div>
           <blockquote class="d-table-cell v-align-top text-gray">
-            <p class="alt-mono-font alt-text-small">“Quote from someone that made their first contribution to open source.”</p>
-            <footer class="alt-text-small">Jane Smith, Columbus OH</footer>
+            <p class="alt-mono-font alt-text-small">Contributing to open source not only allows businesses of every size to take advantage and build upon the talents of others, it’s also an avenue for those new to coding to acclimate themselves to the development process in ways that allow them to practice, experiment, and showcase themselves as valuable and contributing members of the community.</p>
+            <footer class="alt-text-small">Kim Crayton (<a href="https://github.com/kmcrayton7">@kmcrayton7</a>), Community Engineer, Writer, Founder</footer>
```
Comment (MikeMcQuaid) [inline app/views/home/index.html.erb]: @kmcrayton7 Just a heads up that your quote is being used here. Let me know either here or at mikemcquaid@github.com if you have any problem with that. Ignore this if you're 👍. Thanks!

File: app/views/home/index.html.erb [hunk 4]
```diff
@@ -159,25 +159,25 @@
...
         <div>
           <div class="d-table-cell v-align-top pr-3">
-            <img src="https://api.adorable.io/avatars/285/ossfriday2.png" alt="" width="50" height="50" style="border-radius:25px;">
+            <img src="https://avatars1.githubusercontent.com/u/133209?v=3&s=100" alt="" width="50" height="50" style="border-radius:25px;">
           </div>
           <blockquote class="d-table-cell v-align-top text-gray">
-            <p class="alt-mono-font alt-text-small">“Another quote that talks about how Open Source Friday saved my life and my dog.”</p>
-            <footer class="alt-text-small">Juan Perez, Bogotá Columbia</footer>
+            <p class="alt-mono-font alt-text-small">I got to where I am today by contributing to open source software, allowing me to not only improve my skills as a software developer but also to increase my network of colleagues and to meet new and interesting friends from extremely diverse cultures. I can highly recommend it.</p>
+            <footer class="alt-text-small">Chris Lamb (<a href="https://github.com/lamby">@lamby</a>), Freelance Developer, <a href="https://www.debian.org">Debian Project</a> Leader</footer>
```
Comment (MikeMcQuaid) [inline app/views/home/index.html.erb]: @lamby Just a heads up that your quote is being used here. Let me know either here or at mikemcquaid@github.com if you have any problem with that. Ignore this if you're 👍. Thanks!

File: app/views/home/maintainers.html.erb [hunk 5]
```diff
@@ -114,22 +114,22 @@
...
-            <img src="https://api.adorable.io/avatars/285/ossfriday.png" alt="" width="50" height="50" style="border-radius:25px;">
+            <img src="https://avatars1.githubusercontent.com/u/1162160?v=3&s=100" alt="" width="50" height="50" style="border-radius:25px;">
           </div>
           <blockquote class="d-table-cell v-align-top text-gray">
-            <p class="alt-mono-font alt-text-small">“Quote of some wisdom from a maintainer”</p>
-            <footer class="alt-text-small">Jane Smith, Homebrew</footer>
+            <p class="alt-mono-font alt-text-small">The thing that most open source projects lack is not money, but time. Many maintainers have full-time jobs and can't realistically cut back their hours. What almost all projects need is more contributors. Open Source Friday is a perfect way for companies to support the open source ecosystem they depend on in a meaningful way.</p>
+            <footer class="alt-text-small">Rich Harris (<a href="https://github.com/Rich-Harris">@Rich-Harris</a>), <a href="https://github.com/rollup/rollup">Rollup</a> and <a href="https://github.com/sveltejs/svelte">Svelte</a> maintainer</footer>
```
Comment (MikeMcQuaid) [inline app/views/home/maintainers.html.erb]: @Rich-Harris Just a heads up that your quote is being used here. Let me know either here or at mikemcquaid@github.com if you have any problem with that. Ignore this if you're 👍. Thanks!

File: app/views/home/maintainers.html.erb [hunk 6]
```diff
@@ -114,22 +114,22 @@
...
           <div class="d-table-cell v-align-top pr-3">
-            <img src="https://api.adorable.io/avatars/285/ossfriday2.png" alt="" width="50" height="50" style="border-radius:25px;">
+            <img src="https://avatars3.githubusercontent.com/u/54418?v=3&s=100" alt="" width="50" height="50" style="border-radius:25px;">
           </div>
           <blockquote class="d-table-cell v-align-top text-gray">
-            <p class="alt-mono-font alt-text-small">“Another quote that talks about how Open Source Friday improved my project.”</p>
-            <footer class="alt-text-small">Juan Perez, Strap</footer>
+            <p class="alt-mono-font alt-text-small">Once you are a maintainer on a project that others rely on, open source goes from "sharing the code" to "sharing the maintenance". As we all know, "maintenance" takes time which is most effectively spent in a continuous block to help make sure you can finish that e.g. code review before you start to lose context.</p>
+            <footer class="alt-text-small">Brett Cannon (<a href="https://github.com/brettcannon">@brettcannon</a>), <a href="https://github.com/python/cpython">Python core</a> developer</footer>
```
Comment (MikeMcQuaid) [inline app/views/home/maintainers.html.erb]: @brettcannon Just a heads up that your quote is being used here. Let me know either here or at mikemcquaid@github.com if you have any problem with that. Ignore this if you're 👍. Thanks!

File: app/views/home/maintainers.html.erb (hunk 5 above)
Comment (Rich-Harris) [inline app/views/home/maintainers.html.erb]: LGTM

File: app/views/home/index.html.erb (hunk 4 above)
Comment (lamby) [inline app/views/home/index.html.erb]: @MikeMcQuaid  Can you drop "Freelance Developer" entirely? Or put Debian Project Leader first...? Thanks :)

Comment (MikeMcQuaid) [inline app/views/home/index.html.erb]: @lamby Switched in 3e131b2. If you'd much rather I removed Freelance Developer shout and I'll do that.

Comment (lamby) [inline app/views/home/index.html.erb]: @MikeMcQuaid Yeah, thinking about it, please remove "Freelance Developer".

Comment (MikeMcQuaid) [inline app/views/home/index.html.erb]: @lamby Your wish is my command: 6241df8

**Comment by MikeMcQuaid** [comment]
Those with quotes being used here: let me know either here or at mikemcquaid@github.com if you have any problem with it and I'll open a new PR to change it. Ignore this if you're 👍. Thanks!

**Pull request merged**